internal/ingress/controller/store/store.go

```diff
@@ -540,6 +540,10 @@ func (s *k8sStore) extractAnnotations(ing *extensions.Ingress) {
 
 	anns := s.annotations.Extract(ing)
 
+	if anns.SessionAffinity.Type == "cookie" && anns.Rewrite.UseRegex && anns.SessionAffinity.Cookie.Path == "" {
+		glog.Warningf("session-cookie-path should be set when use-regex is true")
+	}
+
```

Comment:
hmm this does not look feel the best place to do validation. Is there no place in controller to do this?

`store` package does not have any information about particular annotations and now you're overloading it with more knowledge whereas controller already has

--

if it is too cumbersome to do this logging in a good way I'd rather not do it